app/models/grouping.rb

```diff
@@ -474,7 +474,6 @@ def decline_invitation(student)
   def deletable_by?(user)
     return false unless self.inviter == user
     (!self.is_valid?) || (self.is_valid? &&
-                          self.accepted_students.size == 1 &&
```

Comment:
I think we should leave this in that students can only delete a group if they are the only member remaining in the group.  Otherwise a student could be blindsided if the inviter deletes the group but doesn't tell them.